website/client/components/chat/chatCard.vue

```diff
@@ -7,7 +7,7 @@ div
       h3.leader(
         :class='userLevelStyle(msg)',
         @click="showMemberModal(msg.uuid)",
-        v-b-tooltip.hover.top="('contributor' in msg) ? msg.contributor.text : ''",
+        v-b-tooltip.hover.top = "('contributor' in msg) ? tierTitle : ''",
```

Comment:
looks like `contributor in msg` is always true so this check can be removed, `v-b-tooltip.hover.top="tierTitle"` will work